src/LaraCart.php

```diff
@@ -557,12 +564,16 @@ public static function formatMoney($number, $locale = null, $internationalFormat
      *
      * @return string
      */
-    public function feeTotals($format = true)
+    public function feeTotals($format = true, $withTax = false)
```

Comment:
since fees cannot have coupons , this should be ok to keep. Because the taxes will always be the same no matter what with fees. You are correct , keep this.